.github/workflows/vib-redis.yaml

```diff
@@ -0,0 +1,75 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    branches:
+      - master
+      - bitnami:master
```

Comment:
Should't it be enough with 'master'?

Reply:
Yes right, not needed. In any case we will work on a full review of the GitHub action & folders structure.